benchmark/micro/pushdown/pwmj_inequality_pushdown.benchmark

```diff
@@ -0,0 +1,21 @@
+# name: benchmark/micro/pushdown/pwmj_inequality_pushdown.benchmark
+# description: Show the performance of using pushdown with PieceWiseMergeJoin
+# group: [pushdown]
+
+name PieceWiseMergeJoinInequalityPushdown
+group micro
+subgroup pushdown
+
+require tpch
+
+cache tpch_sf10.duckdb
+
+load
+CALL dbgen(sf=10);
```

Comment:
Can we use the new way of loading TPC-H I've introduced in https://github.com/duckdb/duckdb/pull/17367?

See e.g. https://github.com/Mytherin/duckdb/blob/907add2cc63ac1c722a91ec655e2fd12d7957177/benchmark/tpch/aggregate/top_n_in_group_agg.benchmark

Reply:
Also fixed NLJ benchmark.